### ISSUE TITLE
Unencoded HTML tags still raise error in 5.5.0-beta.7

### ISSUE THREAD

**Issue by adc-mhaugen** [ember-intl/ember-intl#1443]
Hi.  I'm trying to upgrade from ember-intl@3.5 to ember-intl@5.5.0-beta.7.  I'm using the beta version so I don't have to escape any HTML tags in our translated text.  However, even on the beta version I'm still seeing errors for non-escaped HTML tags.  I've reproduced in a brand new empty project using:

* ember-cli: 3.19.0
* node: 12.18.3
* os: darwin x64

My `en-us.yml` file has two entries:

```yml
price_banner: The {product} costs {price, number, USD}
link: <a href="{href}">Click Here</a>
```

My application template looks like this:

```handelbars
{{t 'link' href="www.google.com" htmlSafe=true}}

{{outlet}}
```

When I build my application I get this error:

```bash
Build Error (TranslationReducer)

An error occurred (Expected "#", "'", "\n", "{", argumentElement, double apostrophes, end of input, or tagElement but "<" found.) when extracting ICU arguments for '<a href="www.google.com">Click Here</a>'
```

Based on the tests added in #1420 I think this should work.  Any advice as to what I'm doing wrong would be greatly appreciated.

**Comment by buschtoens** [comment]
Thank you for reporting this! We think we've identified the issue and will fix it in #1444.

**Comment by buschtoens** [comment]
Released as `5.5.0-beta.8`. 🚀
